debs-to-download

```diff
@@ -21,7 +21,6 @@ newpid
 tcpdump
 gdbserver
 logi-rts
-logiclab
```

Comment:
Please add a reason in the commit message why this package is not installed per default.